microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
+The **Opportunities** section provides specific tips on how to improve the load performance of this specific webpage:
 
-![The Metrics section](../media/speed-glitch-tony-remix-audits-performance-metrics-collapsed-highlighted.msft.png)
+![The Opportunities section](./images/opportunities-section.png)
 
-Click the highlighted toggle button in the following figure to display a description for each metric.  Then click **Learn More** to read documentation about it.
+Click an opportunity to display more information about it, and then click **Learn more** to read documentation about why an opportunity is important, and specific recommendations on how to fix it:
 
-![Click the highlighted toggle button to expand the Metrics items](../media/speed-glitch-tony-remix-audits-performance-metrics-expanded.msft.png)
+![An expanded opportunity, with more information, and a learn more link](./images/expanded-opportunity.png)
 
-Below Metrics is a collection of screenshots that show you how the page looked as it loaded.
+###### Diagnostics
 
-![How the page looked while loading](../media/speed-glitch-tony-remix-audits-performance-view-trace.msft.png)
+The **Diagnostics** section provides more information about factors that contribute to the load time of the page:
 
-The **Opportunities** section provides specific tips on how to improve the load performance of this specific page.
+![The Diagnostics section](./images/diagnostics-section.png)
 
-![The Opportunities section](../media/speed-glitch-tony-remix-audits-performance-view-trace.msft.png)
+###### Passed audits
 
-Click an opportunity to learn more about it.
+The **Passed audits** section shows you what the site is doing correctly.  Click **Show** to expand the section:
 
-![Eliminate render-blocking resources opportunity](../media/speed-glitch-tony-remix-audits-performance-opportunities-expanded.msft.png)
+![The Passed Audits section](./images/passed-audits-section.png)
 
-Click **Learn More** to display documentation about why an opportunity is important, and specific recommendations on how to fix it.
 
-![Documentation for the Eliminate render-blocking resources opportunity](../media/speed-web-dev-performance-audits.msft.png)
+<!-- ====================================================================== -->
+## Step 3: Experiment
 
-The **Diagnostics** section provides more information about factors that contribute to the load time of the page.
+The **Opportunities** section of your report gives you tips on how to improve the performance of the webpage.  In this section, you implement the recommended changes to the codebase, auditing the site again after each change to measure how it affects site speed.
```

Comment:
```suggestion
The **Opportunities** section of your report gives you tips on how to improve the performance of the webpage.  In this section, you implement the recommended changes to the code base, auditing the site again after each change to measure how that change affects site speed.
```
nice to do: Find "code base" vs "codebase" in style guides.  guessing the std is sep words